pex/variables.py

```diff
@@ -319,6 +319,17 @@ def PEX_HTTP_RETRIES(self):
     """
     return self._get_int('PEX_HTTP_RETRIES', default=5)
 
+  # TODO(#94) Remove and push into --flags.
+  @property
+  def PEX_HTTP_TIMEOUT(self):
+    """Integer
+
+    When built with the `requests` HTTP library, a timeout to apply for HTTP connect attempts and
+    read attempts. See the `requests` docis for more information.
```

Comment:
Typo: `docs`. Also I think it would be helpful to point directly to the docs: https://2.python-requests.org/en/master/user/quickstart/#timeouts.